src/vivarium_csu_zenon/globals.py

```diff
@@ -99,13 +99,17 @@ def log_name(self):
 
 
 class __DiabetesMellitus(NamedTuple):
+    PREVALENCE: str = 'cause.diabetes_mellitus.prevalence'
     MODERATE_DIABETES_PREVALENCE: str = 'sequela.moderate_diabetes_mellitus.prevalence'
     SEVERE_DIABETES_PREVALENCE: str = 'sequela.severe_diabetes_mellitus.prevalence'
+    INCIDENCE_RATE: str = 'cause.diabetes_mellitus.incidence_rate'
+    REMISSION_RATE: str = 'cause.diabetes_mellitus.remission_rate'
+    MODERATE_DIABETES_PROPORTION: str = 'sequela.moderate_diabetes_mellitus.proportion'
+    SEVERE_DIABETES_PROPORTION: str = 'sequela.severe_diabetes_mellitus.proportion'
     MODERATE_DIABETES_DISABILITY_WEIGHT: str = 'sequela.moderate_diabetes_mellitus.disability_weight'
     SEVERE_DIABETES_DISABILITY_WEIGHT: str = 'sequela.severe_diabetes_mellitus.disability_weight'
-    MODERATE_DIABETES_EMR: str = 'sequela.moderate_diabetes_mellitus.excess_mortality_rate'
-    SEVERE_DIABETES_EMR: str = 'sequela.severe_diabetes_mellitus.excess_mortality_rate'
     CSMR: str = 'cause.diabetes_mellitus.cause_specific_mortality_rate'
+    EMR: str = 'cause.diabetes_mellitus.excess_mortality_rate'
```

Comment:
You'll need to do one of two things:
1. Set EMRs for each of your "sequelae" states by writing them to the appropriate keys in the artifact (as is done with the disability weights).
2. Use the `get_data_funcs` kwarg on the creation of your sequelae states to point them to this key.

My preference is for option 1.